rollup.config.js

```diff
@@ -18,13 +18,13 @@ const external = (id) => !id.startsWith('.') && !id.startsWith('/') && !id.inclu
 export default [
   {
     input,
```

Comment:
I have an idea to optimize the setup also for Deno...

let's create an index.cjs.js file instead of an index.esm.js file... what do you think? This way the src/index.js would be the correct esm export...

btw: I think the exports: 'defaults' option is not necessary

Reply:
Sure, I don't know anything about deno, but I can try that no problem!

Reply:
please also remove the 'defaults' option here: https://github.com/i18next/i18next/pull/1681/files#diff-6814bf77564b4f1c92f5861e184e28fe217c080a047fefa8b73a728f755ec45cR22

Reply:
done!

Reply:
![image](https://user-images.githubusercontent.com/1086194/141797353-5f248c89-54a5-43c5-bdf2-c40fd5eb6578.png)